src/server/server-quiz/clients/trivia-client.js

```diff
@@ -0,0 +1,36 @@
+const { default: axios } = require("axios");
+
+async function fetchQuestions(url) {
+  const errorResponse = {
+    error: true,
+    description: `Error fetching questions`,
+  };
+  try {
+    const response = await axios.get(url).then((response) => response);
```

Comment:
Mentors always told to use await instead of then

Reply:
I removed the then, I guess it's redundant